crates/frontend/src/utils.rs

```diff
@@ -78,6 +78,16 @@ pub fn get_host() -> String {
             }
         })
         .unwrap_or(String::from("http://localhost:xxxx"));
+    match host.as_str() {
```

Comment:
This can be done inside match js_sys::eval() block, if not can we use if condition instead of match?

Reply:
You can also avoid mutating host variable